conf/base.config

```diff
@@ -89,13 +89,11 @@ process {
   $get_software_versions {
     memory = { check_max( 2.GB, 'memory' ) }
     cache = false
-    executor = 'local'
     errorStrategy = 'ignore'
   }
   $workflow_summary_mqc {
     memory = { check_max( 2.GB, 'memory' ) }
     cache = false
-    executor = 'local'
```

Comment:
Apologies, this process is an exception to the rule - it _can_ set this because it uses `exec` instead of `script`, it doesn't submit any jobs to the cluster.